bin/set-sqfs-links.sh

```diff
@@ -116,6 +116,7 @@ for ncn in "${NCNS_K8S[@]}"; do
     cp -p /var/www/boot/script.ipxe .
     if [[ "$ncn" =~ 'ncn-w' ]]; then
         sed -i -E 's/rd.luks(=1)?\s/rd.luks=0 /g' script.ipxe
+        sed -i -E '/ncn-params .*/ s/$/ split_lock_detect=off/' script.ipxe
```

Comment:
Does this hurt to have on master and storage nodes?

`rd.luks` is special, on workers it must be set to `0` otherwise our codebase creates an etcd partition (which we do not want on worker nodes).
If `split_lock_detect` is safe for all NCNs, then we should move it somewhere more generic.

- https://github.com/Cray-HPE/node-images/blob/main/boxes/sles15-base/http/autoinst.template.xml#L93
- https://github.com/Cray-HPE/metal-ipxe/blob/main/script.ipxe#L32

Reply:
No, it is only needed on NCNs-w's. The only draw back to putting it on all NCNs is that we would not be able to detect other codes than DVS that had an issue with split_lock_detect. If you are ok with that I can move it to the ncn_parms var in the script.ipxe.